templates/coderd.yaml

```diff
@@ -50,7 +54,11 @@ spec:
 {{- include "coder.serviceTolerations" . | indent 6 }}
       initContainers:
         - name: migrations
+          {{- if hasKey .Values "cemanager" }}
           image: {{ .Values.cemanager.image | quote }}
+          {{- else }}
+          image: {{ .Values.coderd.image | quote }}
+          {{- end }}
```

Comment:
This is very messy since we have a lot of values. I think we should just force people to change `cemanager` to `coderd` in this version instead since it's a one line change.

Reply:
I think we need at least 1 version of deprecation before forcing a switch.

Not because changing the value is hard, but customers may have external services that reference `cemanager` for logs and monitoring... us straight-up breaking that without warning sounds sus.

Reply:
In that case we can keep the service and resource names for one version then and change them in the next version